src/main/java/com/codeborne/selenide/SelenideElement.java

```diff
@@ -39,6 +39,29 @@
    */
   SelenideElement setValue(String text);
 
+  /**
```

Comment:
This doc is not assignable to this method.
Use inherit doc or even better just leave block with `@see com.codeborne.selenide.commands.SetValue`
And leave only related method documentstion

Reply:
I hope I did it correctly.